src/invidious/routes/channels.cr

```diff
@@ -28,7 +28,7 @@ module Invidious::Routes::Channels
         end
       end
       items = items.select(&.is_a?(SearchPlaylist)).map(&.as(SearchPlaylist))
-      items.each { |item| item.author = "" }
+      items.each(&.author=(""))
```

Comment:
Wouldn't `&.author = ""` (without the parenthesis) be simpler? Afaik, parenthesis aren't required and just add clutter.